packages/backend/src/controllers/metricsExplorerController.ts

```diff
@@ -45,6 +47,10 @@ export class MetricsExplorerController extends BaseController {
         @Query() compareToMetric?: string,
         @Query() startDate?: string,
         @Query() endDate?: string,
+        @Body()
+        body?: {
+            timeDimensionOverride?: TimeDimensionConfig;
+        },
```

Comment:
body or 3 separate query params instead?

Reply:
I take my words back, body is fine